source/slang/parser.cpp

```diff
@@ -2070,7 +2087,7 @@ namespace Slang
         return decl;
     }
 
-    static void parseOptionalInheritanceClause(Parser* parser, AggTypeDecl* decl)
+    static void parseOptionalInheritanceClause(Parser* parser, ContainerDecl* decl)
```

Comment:
Yeah, this is one place that makes me think that `AssocTypeDecl` should be an `AggTypeDecl`...